.github/workflows/test.yml

```
@@ -303,3 +296,24 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           path: packages/pwa-kit-create-app/local-npm-repo/verdaccio.log
+  lighthouse:
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [16]
```

```suggestion
        node: [16] # Should match node version supported by MRT.
```
Basically [Brian's comment](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1126/files#r1163582246), we should make it explicit _why_ we're choosing node 16 here.